### PR TITLE
Add what is opentelemetry page in french

### PR DESCRIPTION
**Pull request by dmathieu**

### CODE REVIEW

**Comment by svrnm** [comment]
@open-telemetry/docs-fr-approvers @iguitton, @pierrehilbert PTAL!

File: content/fr/docs/what-is-opentelemetry.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+default_lang_commit: 71833a5f8b84110dadf1e98604b87a900724ac33
+---
+
+OpenTelemetry, c'est :
+
+- Un framework
+  d'[observabilité](/docs/concepts/observability-primer/#what-is-observability)
+  et des outils conçus pour créer et gérer des données de télémétrie telles que
+  des [traces](/docs/concepts/signals/traces/),
+  [metriques](/docs/concepts/signals/metrics/), et des
```
Comment (pierrehilbert) [inline content/fr/docs/what-is-opentelemetry.md:15]:
```suggestion
  des [métriques](/docs/concepts/signals/metrics/), et des
```
To be consistent with the previous and next items.

File: content/fr/docs/what-is-opentelemetry.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+L'[observabilité](/docs/concepts/observability-primer/#what-is-observability)
+est la capacité de comprendre l'état interne d'un système en examinant ses
+données sortantes. Dans un contexte logiciel, cela signifie être capable de
+comprendre l'état interne d'un système en examinant ses données de télémétrie,
+qui incluent les traces, métriques et logs.
+
+Pour rendre un système observable, il doit être
+[instrumenté](/docs/concepts/instrumentation). Cela signifie que le code doit
+émettre des [traces](/docs/concepts/signals/traces/),
+[metriques](/docs/concepts/signals/metrics/), et/ou des
```
Comment (pierrehilbert) [inline content/fr/docs/what-is-opentelemetry.md:40]:
```suggestion
des [métriques](/docs/concepts/signals/metrics/), et/ou des
```
To be consistent with the previous and next items.

File: content/fr/docs/what-is-opentelemetry.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+## Composants principaux d'OpenTelemetry
+
+OpenTelemetry se compose des principaux composants suivants :
+
+- Une [spécification](/docs/specs/otel) pour tous les composants
+- Un [protocole](/docs/specs/otlp/) standardisé définissant le format des
+  données de télémétrie
+- Des [conventions sémantiques](/docs/specs/semconv/) définissant une
+  nomenclature standardisé pour décrire les données de télémétrie
```
Comment (pierrehilbert) [inline content/fr/docs/what-is-opentelemetry.md:73]:
```suggestion
  nomenclature standardisée pour décrire les données de télémétrie
```

File: content/fr/docs/what-is-opentelemetry.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+- Des APIs décrivant comment générer des données de télémétrie
+- Des [SDKs par langage](/docs/languages) qui implémentent les spécifications,
+  APIs et l'export des données de télémétrie
```
Comment (pierrehilbert) [inline content/fr/docs/what-is-opentelemetry.md:76]:
```suggestion
  les APIs et l'export des données de télémétrie
```

File: content/fr/docs/what-is-opentelemetry.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+- Un [écosystème de bibliothèques](/ecosystem/registry) qui instrumentent des
+  bibliothèques et frameworks.
+- Des composants d'instrumentation automatique qui génèrent des données de
+  télémétrie sans requérir de modifications de code.
+- Le [Collector OpenTelemetry](/docs/collector), un proxy qui reçoit, modifie et
+  exporte les données de télémétrie
+- Divers autre outils, tel que
```
Comment (pierrehilbert) [inline content/fr/docs/what-is-opentelemetry.md:83]:
```suggestion
- Divers autres outils, tels que
```

File: content/fr/docs/what-is-opentelemetry.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+  l'[Opérateur Kubernetes pour OpenTelemetry](/docs/kubernetes/operator/), les
+  [Helm Charts pour OpenTelemetry](/docs/kubernetes/helm/) et des
+  [ressources communautaires pour les Faas](/docs/faas/)
```
Comment (pierrehilbert) [inline content/fr/docs/what-is-opentelemetry.md:86]:
```suggestion
  [ressources communautaires pour les FaaS](/docs/faas/)
```

File: content/fr/docs/what-is-opentelemetry.md [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+
+OpenTelemetry est utilisé par une grande variété de
+[bibliothèques, services et applications](/ecosystem/integrations/) qui
+intègrent OpenTelemetry afin de fournir de l'observabilité par défaut.
+
+OpenTelemetry is pris en charge par de nombreux
```
Comment (pierrehilbert) [inline content/fr/docs/what-is-opentelemetry.md:92]:
```suggestion
OpenTelemetry est pris en charge par de nombreux
```

File: content/fr/docs/what-is-opentelemetry.md [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+## Extensibilité
+
+OpenTelemetry est conçu pour être extensible. Quelques exemples de la façon dont
+il peut être étendu incluent :
+
+- Ajouter un receveur au Collecteur OpenTelemetry afin de prendre en charge les
+  données de télémétrie venant d'une source de données personnalisée
+- Charger des bibliothèques d'instrumentation personnalisées dans un SDK
+- Créer une [distribution](/docs/concepts/distributions/) d'un SDK, ou du
+  collecteur adaptée à une utilisation spécifique
```
Comment (pierrehilbert) [inline content/fr/docs/what-is-opentelemetry.md:105]:
```suggestion
  collecteur adapté à une utilisation spécifique
```

**Review by pierrehilbert — COMMENTED** [review]
I added some suggestions.
Also, I would choose between adding dots or not at the end of every bullet points to be consistent.

**Comment by pierrehilbert** [comment]
I think you should remove the `lang:pt` from this PR to avoid confusion.
I can't approve the PR but looks great to me, thanks for taking into account my suggestions.
@svrnm Should I follow [this process](https://github.com/open-telemetry/community/issues/new?template=membership.md&title=REQUEST%3A%20New%20membership%20for%20%3Cyour-GH-handle%3E) by adding both you and @dmathieu to be able to approve this kind of PRs?

**Comment by svrnm** [comment]
> I think you should remove the `lang:pt` from this PR to avoid confusion. I can't approve the PR but looks great to me, thanks for taking into account my suggestions. @svrnm Should I follow [this process](https://github.com/open-telemetry/community/issues/new?template=membership.md&title=REQUEST%3A%20New%20membership%20for%20%3Cyour-GH-handle%3E) by adding both you and @dmathieu to be able to approve this kind of PRs?

I'd like to follow the process and stick with the expectations for membership from [our guidelines](https://github.com/open-telemetry/community/blob/main/guides/contributor/membership.md#established-contributors). This means you would need ~5 PRs to be written or reviewed to request membership. The exact number depends on the complexity of the different PRs you helped with.

So what we will do until then is counting your textual reviewes ("lgtm", "looks good to me") as if you have approver status. Just tag me or @open-telemetry/docs-maintainers and one of us can help you with that.

**Pull request merged**